### PR TITLE
zh: update precheck info about MySQL 8.0 support

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added, or deleted? (Required)

1. Update info about MySQL 8.0 support.
2. Link to the precheck info in `migrate-data-using-dm` earlier to make sure the user has information about required privileges when needed.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/627
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: zh/overview.md [hunk 1]
```diff
@@ -41,12 +41,16 @@ DM 支持对源数据的分库分表进行合并迁移，但有一些使用限
...
     - MariaDB 版本 >= 10.1.2
 
     > **注意：**
     >
-    > 如果上游 MySQL/MariaDB server 间构成主从复制结构，则需要 5.7.1 < MySQL 版本 < 8.0 或者 MariaDB 版本大于等于 10.1.3。
+    > 如果上游 MySQL/MariaDB server 间构成主从复制结构，则需要 MySQL 版本大于 5.7.1 或者 MariaDB 版本大于等于 10.1.3。
+
+    > **注意：**
+    >
+    > 支持从 MySQL v8.0 迁移数据是 DM v2.0 的实验特性。
```
Comment (qiancai) [inline zh/overview.md:53]: 这里是否需要像其他实验特性那样，在注意中提到 “不建议在生产环境下使用”。
```suggestion
    > 支持从 MySQL v8.0 迁移数据是 DM v2.0 的实验特性，不建议在生产环境下使用。
```

File: en/precheck.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Upstream and downstream database users must have the corresponding read and writ
 
 + Database version
 
-    - 5.5 < MySQL version
+    - MySQL version > 5.5
```
Comment (qiancai) [inline en/precheck.md:21]: en/overview.md line 108 也需要同样的修改。

Comment (CharLotteiu) [reply en/precheck.md:21]: Updated in https://github.com/pingcap/docs-dm/pull/641/commits/ec0d46394165a2f2075d33f9fd9eec797b875f24.

File: zh/precheck.md [hunk 3]
```diff
@@ -18,9 +18,13 @@ aliases: ['/docs-cn/tidb-data-migration/dev/precheck/']
 
 + 数据库版本
 
-    - 5.5 < MySQL 版本 < 8.0
+    - MySQL 版本 > 5.5
     - MariaDB 版本 >= 10.1.2
 
+    > **注意：**
+    >
+    > 支持从 MySQL v8.0 迁移数据是 DM v2.0 的实验特性。
```
[the remark this answers is not in the document]
Comment (CharLotteiu) [reply zh/precheck.md]: Agreed. Updated in https://github.com/pingcap/docs-dm/pull/641/commits/e76c4c9d5540599d1a3f03a6fdebbfb70258b3d6.

File: zh/overview.md [hunk 4]
```diff
@@ -41,12 +41,16 @@ DM 支持对源数据的分库分表进行合并迁移，但有一些使用限
...
-    - 5.5 < MySQL 版本 < 8.0
+    - MySQL 版本 > 5.5
     - MariaDB 版本 >= 10.1.2
 
     > **注意：**
     >
-    > 如果上游 MySQL/MariaDB server 间构成主从复制结构，则需要 5.7.1 < MySQL 版本 < 8.0 或者 MariaDB 版本大于等于 10.1.3。
+    > 如果上游 MySQL/MariaDB server 间构成主从复制结构，则需要 MySQL 版本大于 5.7.1 或者 MariaDB 版本大于等于 10.1.3。
+
+    > **注意：**
```
[the remark this answers is not in the document]
Comment (CharLotteiu) [reply zh/overview.md]: See https://github.com/pingcap/docs-dm/pull/641/commits/e76c4c9d5540599d1a3f03a6fdebbfb70258b3d6.

**Pull request merged**